tempest-testing-internal/src/main/kotlin/app/cash/tempest/testing/DefaultTestDynamoDbClient.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.cash.tempest.testing
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBStreams
+import com.google.common.util.concurrent.AbstractIdleService
+
+class DefaultTestDynamoDbClient(
```

Comment:
Documentation?

Reply:
In particular, what’s the difference between this and the TestDynamoDbClient base class?

Reply:
TestDynamoDbClient is just the interface?